src/github.js

```diff
@@ -69,6 +69,28 @@ export default function githubFactory(token, repository) {
                 });
             });
         },
+        /**
+         * Create the release pull request
+         * @param {String} releasingBranch - the temp branch that contains the commits to release
+         * @param {String} releaseBranch - the base branch
+         * @param {String} version - the version of the release
+         * @param {String} fromVersion - the last version
+         * @returns {Promise<Object>} - resolves with the pull request data
+         */
+
```

Comment:
Blank line to remove.

Also, none of the JSDoc above matches the function beneath it (I guess the blank line was doing some work here...)

Reply:
I created this PR, since the original one did not work, @edwin-focaloid , I am sure you can continue from here and with this PR, let me know if you need my help

Reply:
updated